captum/attr/_core/noise_tunnel.py

```diff
@@ -43,13 +46,13 @@ def __init__(self, attribution_method):
 
     def attribute(
         self,
-        inputs,
-        nt_type="smoothgrad",
-        n_samples=5,
-        stdevs=1.0,
-        draw_baseline_from_distrib=False,
+        inputs: Union[Tensor, Tuple[Tensor, ...]],
+        nt_type: str = "smoothgrad",
+        n_samples: int = 5,
+        stdevs: Union[float, Tuple[float, ...]] = 1.0,
+        draw_baseline_from_distrib: bool = False,
         **kwargs
-    ):
+    ) -> Union[Tensor, Tuple[Tensor]]:
```

Comment:
This should include Tuple[Tensor, ...] not just Tuple[Tensor] to account for tuples with multiple tensors.